### PR TITLE
Porting denormals document proofreading #12288

### PR DESCRIPTION
**Pull request by sgolebiewski-intel**
Porting changes from
https://github.com/openvinotoolkit/openvino/pull/12288/

### CODE REVIEW

File: pages/openvino_docs_OV_UG_supported_plugins_CPU.rst [hunk 1]
```diff
@@ -631,7 +374,10 @@ The plugin supports the following properties:
 Read-write Properties
```
Comment (sgolebiewski-intel) [inline pages/openvino_docs_OV_UG_supported_plugins_CPU.rst:374]: I am thinking about placing both "Read-write Properties" and "Read-only Properties" in two tabs.
What do you think guys?

**Review by kblaszczak-intel — APPROVED** [review]
That is a hell of an article to port O_O I haven't found anything suspicious, but we may want to check it again in the future :D

File: pages/openvino_docs_OV_UG_supported_plugins_CPU.rst (hunk 1 above)
Comment (kblaszczak-intel) [reply pages/openvino_docs_OV_UG_supported_plugins_CPU.rst:374]: Let's write it down as a planned change and leave for future revisions.

**Comment by sgolebiewski-intel** [comment]
> That is a hell of an article to port O_O I haven't found anything suspicious, but we may want to check it again in the future :D

Yeah, took me some time, but managed to get rid of ".. raw:: html " completely. Have tested it locally, seems legit :D

File: pages/openvino_docs_OV_UG_supported_plugins_CPU.rst (hunk 1 above)
Comment (sgolebiewski-intel) [reply pages/openvino_docs_OV_UG_supported_plugins_CPU.rst:374]: Sure thing, will do 😄

**Pull request merged**